<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../ventanas/imagenes/126620-illustrazione-del-gabbia-toracica-gratuito-vettoriale-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>461</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>imagenes/smooth-stucco-wall_1194-7087.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="text">
    <string>VOLVER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="examinar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="text">
    <string>Examinar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 5, 14);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Examinar archivo NIFTI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_nifti">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>119</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

border-radius:10px</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
